generators/app/templates/src/constants/routes.js

```diff
@@ -0,0 +1,6 @@
+// TODO Add this if you need it
+// export const DOWNLOAD_APP = '/downloadapp';
+
+export const HOME = '/';
```

Comment:
I'd export as default a object to avoid apply destructuring when the routes are imported in a file.